### PR TITLE
README.md Update

### PR DESCRIPTION
**Pull request by shivam-sharma7**
README.md Update

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1532"></a>

### CODE REVIEW

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
You've made a separate PR for updating bio #1531, so this PR should only contain `README.md` file! 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Check out the website: https://linkfree.eddiehub.io
 
 ## 👇 Prerequisites
 
-Before installation, please make sure you have already installed the following tools:
+Before Contribute or add new feature, please make sure you have already installed the following tools:
```
Comment (Panquesito7) [inline README.md:32]: Spelling errors here.
```suggestion
Before contributing or adding a new feature, please make sure you have already installed the following tools:
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Check out the website: https://linkfree.eddiehub.io
 
 ## 👇 Prerequisites
 
-Before installation, please make sure you have already installed the following tools:
+Before Contributing or adding new feature, please make sure you have already installed the following tools:
```
Comment (Panquesito7) [inline README.md:32]: There's a missing `a` here.
```suggestion
Before contributing or adding a new feature, please make sure you have already installed the following tools:
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

File: README.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ Check out the website: https://linkfree.eddiehub.io
 
 ## 👇 Prerequisites
 
-Before installation, please make sure you have already installed the following tools:
+Before Contributing or adding a new feature, please make sure you have already installed the following tools:
```
Comment (Panquesito7) [inline README.md:32]: Hmm, I'm not sure if we should capitalize `contributing` here.
What do you all think about this?


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Comment (shivam-sharma7) [reply README.md]: Yeah you are correct

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Review by kumarsonsoff3 — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Pull request merged**